### ISSUE TITLE
[Documentation] Examples code from scratch 

### ISSUE THREAD

**Issue by luist1228** [clauderic/dnd-kit#966]
The stories code is not very helpful and going to the code in the main repo doesn't help either, I have some doubts about how to implement a kanban like Dnd example and the stories examples are great but I cannot fully grasp the concepts from neither from the doc option in story nor the repo code.

**Comment by Jiert** [comment]
I completely agree. When I want to implement something like the "Sortable Multiple Containers Basic Setup" and look at the code, the only thing I see is 

```
() => <MultipleContainers />
```

https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/docs/presets-sortable-multiple-containers--basic-setup

What am I missing?

**Comment by notrealdev** [comment]
same here, the art of concealment?

**Comment by marcus-wishes** [comment]
Yes, I agree on that. 
I am trying to debug something, and it's really difficult to get a codesandbox.io example going, when the components and elements are distributed over several places, and have properties which are of no use to the current story.
I am fine with the stories, I also use storybook. But I try to keep the stories mainly self sufficient. That also enables the show code option in the docs part as well.

**Comment by sadikyalcin** [comment]
The docs here are probably the least helpful docs I've ever seen.

**Comment by luist1228** [comment]
@Jiert The problem is that the storybook is just a cupple of massive components that depending on the example changes the props for the setup, or at least that's my understanding by checking the storybook code. The only option available is to navigate the examples folder try to understand the implementation navigating the code :v

**Comment by hisnameisjimmy** [comment]
The library looks like it produces beautiful output, but the amount of code necessary for the most simple sortable list is kind of astounding. I look at the Sandbox example and am overcome with feelings of "Is there any way I can avoid drag and drop entirely".

**Comment by hisnameisjimmy** [comment]
I have created a really simple repo for a sortable vertical list, which is what I was going for initially. They actually have decent sample code down in Sortable: https://docs.dndkit.com/presets/sortable

But if you try to run it, there's no styling so nothing will show on the screen. Confusing! I added some simple TailwindCSS classes to make it show up and look obvious. Hopefully this is useful to other people like me who just wanted something dead simple, but instead found sample projects in the Code Sandbox that contained many files and confusing to read.

**Comment by akshay-nm** [comment]
> I have created [a really simple repo for a sortable vertical list](https://github.com/hisnameisjimmy/simple-sortable-list-dnd-kit), which is what I was going for initially. They actually have decent sample code down in Sortable: https://docs.dndkit.com/presets/sortable
> 
> But if you try to run it, there's no styling so nothing will show on the screen. Confusing! I added some simple TailwindCSS classes to make it show up and look obvious. Hopefully this is useful to other people like me who just wanted something dead simple, but instead found sample projects in the Code Sandbox that contained many files and confusing to read.

This is for single sortable list. Could you also try to setup a Kanban?

**Comment by akshay-nm** [comment]
> I completely agree. When I want to implement something like the "Sortable Multiple Containers Basic Setup" and look at the code, the only thing I see is
> 
> ```
> () => <MultipleContainers />
> ```
> 
> https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/docs/presets-sortable-multiple-containers--basic-setup
> 
> What am I missing?

Here is a link to the Component being used in the storybook.
https://github.com/clauderic/dnd-kit/blob/master/stories/2%20-%20Presets/Sortable/MultipleContainers.tsx

**Comment by mattbf** [comment]
+1 Would also love to see simple sandboxes, much easier to learn / reproduce than trying to dig through the repo

**Comment by martrohelsanchez** [comment]
This library has one of the worst documentation I've ever seen. So many things are hidden and not documented.

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
